yarn-project/accounts/src/ecdsa/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from './ecdsa_k/index.js';
+export * from './ecdsa_r/index.js';
```

Comment:
Would've been nice to have all of the ecdsa_r account changes happen in a PR separate from the benchmark stuff, since those are ultimately unrelated.